### PR TITLE
Add cloudwatch_logs c plugin to cloudwatch example

### PR DESCRIPTION
**Pull request by PettitWesley**
Signed-off-by: Wesley Pettit <wppttt@amazon.com>

*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+			"memoryReservation": 50
+		},
+		{
+			 "essential": true,
+			 "image": "nginx",
+			 "name": "app",
+			 "logConfiguration": {
+				"logDriver":"awsfirelens",
+				"options": {
+					"Name": "cloudwatch",
```
Comment (matthewfala) [inline examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json]: the title of this file is "cloudwatch_logs" however it seems we are using "cloudwatch" which may be the go plugin.

File: examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
+	"family": "firelens-example-cloudwatch",
```
Comment (matthewfala) [inline examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json]: consider revising to firelens-example-cloudwatchlogs

**Review by matthewfala — CHANGES_REQUESTED** [review]
c, go, plugin flavor issue with the revised task-definition

File: examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json (hunk 1 above)
Comment (PettitWesley) [reply examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json]: lol oops

File: examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json (hunk 2 above)
Comment (PettitWesley) [reply examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json]: thanks/fixed

File: examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json (hunk 1 above)
Comment (PettitWesley) [reply examples/fluent-bit/cloudwatchlogs/task-definition-cloudwatch_logs.json]: thanks/fixed.

**Review by matthewfala — APPROVED** [review]
Looks good!

**Pull request merged**